### DISCUSSION TITLE
NextJS easter egg UwU page

### DISCUSSION THREAD

**Discussion by wiserat** [vercel/next.js#65230] [Ideas]
### Goals

Other frameworks including React, Vue, Hono, and so on have a secret UwU landing page. You can open it via query parameters. 
https://vuejs.org/?uwu
https://react.dev/?uwu=true

### Background

NextJS needs this feature. It's cool. A lot of NextJS users will feel like their favorite framework finally understands them.

### Proposal

Just load a new UwU landing page when the query parameter is set. I could contribute for sure. I use NextJS daily.

**Comment by Netail** [comment]
https://nextjs.org/?uwu

**Reply by wiserat** [reply]
they added it literally two days ago

**Reply by Netail** [reply]
correct
